dev_guide/routes.adoc

```diff
@@ -225,14 +225,16 @@ ifdef::openshift-online[]
 [[custom-route-and-hosts-and-certificates-restrictions]]
 == Restrictions
 
-Custom route hosts are not permitted. Instead, the following host template is
-enforced on all user routes:
+Routes are restricted in {product-title} Starter, but are not restricted in
+{product-title} Pro. Custom route hosts are permitted in {product-title} Pro. If
+using {product-title} Starter, the following host template is enforced on all
+user routes:
 
 ----
 <route-name>-<namespace>.1d35.starter-us-east-1.openshiftapps.com
 ----
 
-Also, custom certificates are not permitted. Only unencrypted routes, edge
+Custom certificates are not permitted. Only unencrypted routes, edge
```

Comment:
I think this is also incorrect. Custom certificiates *should* work on the Pro tier. @tiwillia Can you confirm?

Reply:
@wgordon17 I'm fairly certain we do allow custom certificates in pro tier. When creating a route, custom certificates can be uploaded. This will need to be changed.